code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -772,20 +772,24 @@
 	wrenchable = FALSE
 	isshared = TRUE
 	products = list(
+		"Pill Packet" = list(
+			/obj/item/storage/pill_bottle/packet/bicaridine = -1,
+			/obj/item/storage/pill_bottle/packet/kelotane = -1,
+			/obj/item/storage/pill_bottle/packet/tramadol = -1,
+			/obj/item/storage/pill_bottle/packet/tricordrazine = -1,
+			/obj/item/storage/pill_bottle/packet/dylovene = -1,
+			/obj/item/storage/pill_bottle/packet/peridaxon = -1,
+			/obj/item/storage/pill_bottle/packet/quickclot = -1,
+			/obj/item/storage/pill_bottle/packet/paracetamol = -1,
+			/obj/item/storage/pill_bottle/packet/hypervene = -1,
+		),
 		"Pill Bottle" = list(
-			/obj/item/storage/pill_bottle/bicaridine = 12,
-			/obj/item/storage/pill_bottle/kelotane = 12,
-			/obj/item/storage/pill_bottle/tramadol = 12,
-			/obj/item/storage/pill_bottle/tricordrazine = 12,
-			/obj/item/storage/pill_bottle/inaprovaline = 12,
-			/obj/item/storage/pill_bottle/dexalin = 12,
-			/obj/item/storage/pill_bottle/dylovene = 12,
-			/obj/item/storage/pill_bottle/spaceacillin = 12,
-			/obj/item/storage/pill_bottle/alkysine = 12,
-			/obj/item/storage/pill_bottle/imidazoline = 12,
-			/obj/item/storage/pill_bottle/peridaxon = 8,
-			/obj/item/storage/pill_bottle/quickclot = 8,
-			/obj/item/storage/pill_bottle/hypervene = 8,
+			/obj/item/storage/pill_bottle/inaprovaline = -1,
+			/obj/item/storage/pill_bottle/dexalin = -1,
+			/obj/item/storage/pill_bottle/spaceacillin = -1,
+			/obj/item/storage/pill_bottle/alkysine = -1,
+			/obj/item/storage/pill_bottle/imidazoline = -1,
+			/obj/item/storage/pill_bottle/hypervene = -1,
```

Comment:
i disagree with having all of those as pill bottles, i think the choice you gave as pill packet was fine.

I think it's missing innapro and dexalin in pill packet, but remove those bottles

Reply:
As I said on discord, I think these being pill bottles are fine as they're entirely utility medication and I think having more of them in a marines kit is fine as you trade a magazine or two for it.